audinterface/core/process.py

```diff
@@ -411,7 +415,16 @@ def process_files(
             task_description=f'Process {len(files)} files',
         )
         self.verbose = verbose
-        return pd.concat(y)
+
+        y = list(itertools.chain.from_iterable([x[0] for x in xs]))
```

Comment:
Why is 

```python
y = [x[0] for x in xs]
```
not working?

Reply:
We have a list of lists that we want to flatten.

Reply:
Ah, ok, so `x` is not a tuple containing single `(y, file, start, end)` entries, but `(y, files, starts, ends)` lists, correct?

Reply:
yes